.github/workflows/publish-preview.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish a preview build
+
+on:
+  issue_comment:
+    types: created
+
+jobs:
+  publish-preview:
+    permissions:
+      contents: write
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body, '@metamaskbot publish-preview') }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.sha }}
```

Comment:
Apparently the SHA for this event trigger is "Last commit on default branch", so this is wrong.

The [event](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#issue_comment) has the PR number. I guess we have to derive the commit hash from that. Unless we find another suitable trigger.

Reply:
Fixed in https://github.com/MetaMask/controllers/pull/965/commits/d12e403f0ab4dd63dc8b134048ad5593179b042e